AirCasting/AuthViews/UserSettings.swift

```diff
@@ -16,8 +17,20 @@ class UserSettings: ObservableObject {
         }
     }
     
+    var keepScreenOn: Bool {
```

Comment:
wasn;t this already merged in in different PR? Some git issue perhaps on your side?